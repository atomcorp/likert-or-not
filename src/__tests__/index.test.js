import React from 'react';
import LikertOrNot from '../index';
import renderer from 'react-test-renderer';

test('LikertOrNot loads', () => {
  const component = renderer.create(<LikertOrNot
    title="The title"
    labels={['Good', 'OK', 'Bad']}
    rows={[
      {
        statement: 'This is a question, how do you feel about it?',
      },
    ]}
    handleClick={(e, { id, value }) => ({})}
  />)
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})