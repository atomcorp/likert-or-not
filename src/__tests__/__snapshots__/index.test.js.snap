// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LikertOrNot loads 1`] = `
<table
  className={undefined}
>
  <caption>
    The title
  </caption>
  <thead
    className={undefined}
  >
    <tr
      className="undefined null"
    >
      <td
        className={undefined}
      >
        Good
      </td>
      <td
        className={undefined}
      >
        OK
      </td>
      <td
        className={undefined}
      >
        Bad
      </td>
    </tr>
  </thead>
  <tbody
    className={undefined}
  >
    <tr
      className="undefined null"
    >
      <td
        className={undefined}
      >
        This is a question, how do you feel about it?
      </td>
      <td>
        <input
          checked={false}
          className={undefined}
          name={undefined}
          onChange={[Function]}
          type="radio"
          value={0}
        />
      </td>
      <td>
        <input
          checked={false}
          className={undefined}
          name={undefined}
          onChange={[Function]}
          type="radio"
          value={1}
        />
      </td>
      <td>
        <input
          checked={false}
          className={undefined}
          name={undefined}
          onChange={[Function]}
          type="radio"
          value={2}
        />
      </td>
    </tr>
  </tbody>
</table>
`;
